spec/models/offer_condition_spec.rb

```diff
@@ -9,6 +9,14 @@
     end
   end
 
+  describe 'touching' do
+    it 'changes the updated at timestamp on the offer' do
```

Comment:
Can we add a test that ensures the application choice is updated too?

I'm not 100% sure but I don't think touching the offer will then also touch the application choice. We may need to move the `touch: true` to the application choice relation if it doesn't work

Reply:
100% it does also touch the application choice, ill add an additional check as proof.